.changeset/tame-pots-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-theming": minor
+---
```

Comment:
Just curious - does this need two changesets?

Reply:
🤔 probably not, I was planning to only add one thing, but then decided to send to commits into the same PR and that caused pushing both files. The good news is that changesets should be able to cover that.